styles.css

```diff
@@ -2,6 +2,250 @@
     box-sizing: border-box;
 }
 
-body {
-    background-color: rgb(234, 234, 234);
+body{
+    /* background-color: black; */
+    color: aliceblue;
+    height: 250vh;
```

Comment:
you cannot set the height of the body to 250vh. Since there will be sections written by javid as well. So when we merge 250vh wont be enough to accomodate all content. 

You should think carefully before writing Global code that can affect the work of other developers or when the work of all developers are merged. You can do whatever you want if the code that you write is local.

Reply:
changed the height from body section to home-page